tests/driver_xbee/Makefile

```diff
@@ -9,7 +9,6 @@ BOARD_INSUFFICIENT_MEMORY := nucleo32-f031 nucleo32-f042 nucleo-f030 nucleo-f334
 USEMODULE += xbee
 USEMODULE += gnrc_netif
 USEMODULE += gnrc_txtsnd
-USEMODULE += gnrc_netdev
```

Comment:
Now it builds but ends up after in a kernel panic:
```
2017-11-14 10:52:17,684 - INFO # 0x3b83
2017-11-14 10:52:17,686 - INFO # *** RIOT kernel panic:
2017-11-14 10:52:17,687 - INFO # FAILED ASSERTION.
2017-11-14 10:52:17,688 - INFO # 
2017-11-14 10:52:17,688 - INFO # *** halted.
2017-11-14 10:52:17,689 - INFO #
```

Reply:
That is not a kernel panic, but again a failed assertion. It would be really helpful if you would compile with `CFLAGS=-DDEBUG_ASSERT_VERBOSE`, because that would give you file and line of the failed assertion ;-)

Reply:
I have this:
```sys/net/gnrc/netif2/gnrc_netif2.c:1075 => 0xbbd
2017-11-14 11:00:41,442 - INFO # *** RIOT kernel panic:
2017-11-14 11:00:41,443 - INFO # FAILED ASSERTION.
```

Reply:
for both `driver_xbee` and `gnrc_networking` applications

Reply:
Thanks! Got my hands on an XBee now, will see if I can find the issue